models/tmp/stg_stripe__invoice_line_item_tmp.sql

```diff
@@ -2,3 +2,5 @@
 
 select * 
 from {{ var('invoice_line_item') }}
+
+{{ livemode_predicate() }}
```

Comment:
It looks like our ERD shows `livemode` as `live_mode`. Let's confirm with the PM what the is accurate here and if we should make an adjustment to the ERD.

![image](https://user-images.githubusercontent.com/74217849/129411336-51fe3bb7-2b86-4f7b-a714-b628f2601c53.png)

Reply:
yeah pretty sure the ERD is incorrect -- we've been using `livemode` in the columns [macro](https://github.com/fivetran/dbt_stripe_source/blob/master/macros/get_invoice_line_item_columns.sql#L11)

Reply:
Good call! Looks good then!